packages/landing/components/QandA/QandA.tsx

```diff
@@ -0,0 +1,37 @@
+"use client";
+
+import { Icon } from "@/ui";
+import { cx } from "class-variance-authority";
+import { PropsWithChildren, useState } from "react";
+
+interface QandAProps extends PropsWithChildren {
+  q: string;
+}
+
+export const QandA = ({ q, children }: QandAProps) => {
```

Comment:
Maybe call it `QuestionAccoridon` or `Q&AAccoridon` or something similar. I feel this name is too vague.

Reply:
Q&A usually represents question and answer, that I feel represents what this is better than the suggestions you gave.

Reply:
I just think it's too generic. We can give it a little more context with Accordion.

Reply:
And maybe `QAndAAccordion`. QandA it's a weird pattern

Reply:
I'll take the `QAndAAccordion`